README.md

```diff
@@ -20,11 +20,14 @@ gem "lita-google"
 
 * `safe_search` (String, Symbol) - The safe search setting to use. Possible values are `:active`, `:moderate`, and `:off`. Default: `:active`.
 
+* `excluded_domains` (Array) - Domains from which you never want to see results. It should be an array of domain strings.
```

Comment:
Mark the type as `Array<String>` and then the second sentence isn't necessary. "Domains to filter out of the results" would also be better phrasing for the first sentence.